.github/workflows/generate-pinval.yaml

```diff
@@ -132,17 +116,13 @@ jobs:
         uses: astral-sh/setup-uv@v5
         with:
           enable-cache: true
-          cache-dependency-glob: uv.lock
+          cache-dependency-glob: scripts/generate_pinval/uv.lock
```

Comment:
I noticed that this cache was never getting hit because `cache-dependency-glob` is defined relative to the root of the repo, so the action was ignoring the cache. This fix causes the cache to save and load correctly; see [this log line](https://github.com/ccao-data/pinval/actions/runs/15744079605/job/44376383575#step:4:27) for evidence of a cache hit.